guides/common/assembly_installing-capsule-server.adoc

```diff
@@ -37,6 +37,19 @@ ifdef::katello,satellite[]
 == Configuring {SmartProxyServer} with SSL Certificates
 endif::[]
 
+ifeval::["{build}" == "foreman"]
```

Comment:
I think that you can use a simpler condition here to make it more readable, I'm guessing `ifdef::foreman-deb[]`.
@melcorr Can you please confirm that this suggestion is correct?

Reply:
AFAICT yep. This is probably leftover of super old text somewhere...